src/frontend/components/search/Search.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
+import styled from 'styled-components'
+import { searchWordSubmitted } from './searchSlice'
+
+const SearchContainer = styled.div`
+  padding: 0 20px;
+
+  form {
+    display: flex;
+  }
+
+  input {
+    width: 75%;
+    margin-right: 5%;
+  }
+
+  button {
+    width: 20%;
+  }
+`
+
+export const Search = () => {
+  const [value, setValue] = useState('')
+  const dispatch = useDispatch()
+  const disabled: boolean = !value.length
+
+  const handleSubmit = (e: any) => {
```

Comment:
This should be `(e: React.SyntheticEvent)`

https://reactjs.org/docs/events.html